registrations/api.py

```diff
@@ -417,7 +417,12 @@ class SignUpViewSet(
 ):
     serializer_class = SignUpSerializer
     queryset = SignUp.objects.all()
-    filter_backends = [SignUpFilterBackend]
+    filter_backends = [
+        SignUpFilterBackend,
+        filters.OrderingFilter,
+    ]
+    ordering_fields = ("first_name", "last_name")
+    ordering = ("first_name", "last_name")
```

Comment:
Nice that you changed the previous `queryset = SignUp.objects.all().order_by("first_name", "last_name")` solution to the current solution of using OrderingFilter.